src/client.js

```diff
@@ -541,6 +543,61 @@ class Client {
          * @type {Campaign.Application}
          */
         this.application = null;
+
+        // temporary start in 2000
+        this.lastTime = "2000-01-01T00:00:00.000";
```

Comment:
Let's review the design and implement this feature in a separate object (CacheRefresher) which can be toggled on/off.